Realm/RLMObjectBase.h

```diff
@@ -33,6 +33,9 @@ RLM_ASSUME_NONNULL_BEGIN
 
 + (NSString *)className;
 
+/// Returns whether the class is included in the default set of classes persisted in a Realm.
```

Comment:
I don't think this should be part of the public interface IMO. If a user wants custom schemas, let them use custom schemas. The whole point of default configurations is to avoid additional default configuration state like this.